torch/utils/benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -743,13 +743,13 @@ def check_result(completed_process):
             # =============================================================================
             # == User code block ==========================================================
             # =============================================================================
-            callgrind_bindings.valgrind_toggle()
+            callgrind_bindings._valgrind_toggle()
```

Comment:
can you split the valgrind change into it's own PR?

Reply:
Sure